parsons/utilities/sql_helpers.py

```diff
@@ -12,3 +12,17 @@ def redact_credentials(sql):
     sql_censored = re.sub(pattern, "CREDENTIALS REDACTED", sql, flags=re.IGNORECASE)
 
     return sql_censored
+
+
+def get_sql_from_file(sql_file):
```

Comment:
Is this function really worth having? It's just wrapping f.read. I think a code-smell here is that this could be reading _any_ file, from a jpg or a blog post, not just a SQL file. That would mean this function isn't adding value beyond just using the `File.reader()` method.